rethinkdb/Dockerfile

```diff
@@ -4,15 +4,13 @@ FROM gitpod/workspace-full
 
 # Install RethinkDB build dependencies.
```

Comment:
RethinkDB is deprecated and I think this Dockerfile doesn't even get built anymore.

Please delete the entire `rethinkdb` directory.

Reply:
done